src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    public void setRadius(double radius) {
+        if (radius > 0) {
+            this.radius = radius;
+        }
+    }
```

Comment:
```suggestion
    public Circle(double radius, Color color) {
        super(color);
        this.radius = radius;
    }
```
same for rest of the child figures - create constructors for them

Reply:
add